ci-runner/helper/GitManager.go

```diff
@@ -99,8 +101,16 @@ func NewGitManagerImpl(gitCliManager GitCliManager) *GitManager {
 	}
 }
 
+func ChangeOrderOfExecutionOfMaterialForCloning(ciProjectDetails []CiProjectDetails) {
```

Comment:
it should be named based on its true  functionality ie: `sortBYcheckoutpath` and at call sight explain the impact of this sorting in comments.
this makes the function more generic

Reply:
Resolved